src/index.html

```diff
@@ -1,13 +1,178 @@
+        <h2 class="info__heading-title">Miami best</h2>
+        <h2 class="info__heading-title">Real estate <a href="#">here</a></h2>
+      </div>
+      <div class="info__text">
+        <h3 class="info__text-title">Over 50 years experience in love.</h3>
+        <p>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+        <p>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.</p>
+        <p class="info__text-indent"><strong>We will never be late!</strong></p>
+        <p class="info__text-indent"><strong>Guaranteed secure transaction - 100%</strong></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="section">
+    <h2 class="heading section__heading" id="gallery">Gallery<span class="heading__crown"></span></h2>
+    <div class="gallery">
+      <div class="photos photos_wide_left">
+        <img src="./images/photos/1.jpg" alt="" class="photos__img">
+        <img src="./images/photos/2.jpg" alt="" class="photos__img">
+      </div>
+      <p class="gallery__captions"><a href="#">Marina Palms</a> / North Miami Beach, FL 33162</p>
+    </div>
+    <div class="gallery">
+      <div class="photos">
+        <img src="./images/photos/3.jpg" alt="" class="photos__img">
+        <img src="./images/photos/4.jpg" alt="" class="photos__img">
+      </div>
+      <p class="gallery__captions"><a href="#">Edition Residens</a> / Miami Beach, FL 33139</p>
+    </div>
+
+    <div class="gallery">
+
+      <div class="photos photos_wide_right">
+        <img src="./images/photos/5.jpg" alt="" class="photos__img">
+        <img src="./images/photos/6.jpg" alt="" class="photos__img">
+      </div>
+
+      <p class="gallery__captions"><a href="#">Faena House</a> / Miami Beach, FL 33140</p>
+
+      <button class="button gallery__button">View All</button>
+    </div>
+    </section>
+
+  <section class="section">
+    <h2 class="heading section__heading" id="consultation">Consultation<span class="heading__crown"></span></h2>
+    <div class="consultation">
+
+      <div class="consultation__text">
+        <h class="consultation__heading">How long have you do<br> yourself a gift?</h>
+        <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+      </div>
+
+      <form class="form consultation__form">
+        <label><input class="input form__input" type="text" name="name" required placeholder="Name"></label>
```

Comment:
Переноси длинные строки
```suggestion
        <label>
          <input 
            class="input form__input" 
            type="text" 
            name="name" 
            placeholder="Name"
            required
          >
        </label>
```